addon/visit.js

```diff
@@ -221,7 +214,7 @@ class PageVisitor {
     async processAtEndOfRun() {
     }
 
-    // This is used to get the viewport size if it is not in the PageVisitor's form.
+    // This is used to get the viewport size.
     async getViewportHeightAndWidth() {
```

Comment:
For consistency with formOptions(), I'd either comment this out or make both of these abstract methods raise an error. Your choice.

Reply:
I'd prefer to got the abstract route so `visitAllPages()` isn't calling methods it doesn't know about.

Reply:
sgtm. That's usually what I do in Python. I was just lazy here.

Reply:
Cool! I'll resolve and merge.